tests/runtime/servicers/test_global_predict_servicer_impl.py

```diff
@@ -203,6 +203,33 @@ def req_iterator() -> Iterator[predict_class]:
     assert count == 100
 
 
+@pytest.mark.parametrize(
+    "good_model_path", ["tests/fixtures/models/foo-bidi-streaming"]
```

Comment:
It doesn't look like this needs to be parametrized?

Reply:
hmm. yeah.. I'll remove that and make a proper fixture